### PR TITLE
Wg add currency selection to thist

### PR DESCRIPTION
**Pull request by wkg7vb**
added currency selection to homepage, will do trends page when the pr is pushed later.
also reordered transaction history to match main

### CODE REVIEW

**Comment by max-dejesus** [comment]
one-line solution. just pull in user locale as expected : line 63
image

**Comment by wkg7vb** [comment]
should be good to approve now

File: BAR/Components/Pages/Transactions/TransactionHistory.razor [hunk 1]
```diff
@@ -31,29 +32,29 @@
...
+    Responsive="true"
+    GridSettingsChanged="OnGridSettingsChanged"
+    SettingsProvider="GridSettingsProvider">
         <GridColumns>
             <GridColumn TItem="UserTransaction" HeaderText="Date" SortKeySelector="item => item.TransactionDateTime" IsDefaultSortColumn="true" SortDirection="SortDirection.Descending">
-                @context.TransactionDateTime.ToShortDateString()
-            </GridColumn>
-            <GridColumn TItem="UserTransaction" HeaderText="Category" SortKeySelector="item => item.TransactionCategory">
-                @context.TransactionCategory
+                <text>@context.TransactionDateTime.ToShortDateString()</text>
```
Comment (max-dejesus) [inline BAR/Components/Pages/Transactions/TransactionHistory.razor]: didn't you want to remove this

**Review by max-dejesus — APPROVED** [review]
ty

**Pull request merged**